astropy/constants/__init__.py

```diff
@@ -68,10 +74,10 @@ def set_enabled_constants(modname):
     import warnings
     from . import utils as _utils
 
-    # NOTE: Update this when default changes.
+    # NOTE: Update this whenever versions are added.
     if modname == 'astropyconst13':
-        from .astropyconst13 import codata2010 as codata
-        from .astropyconst13 import iau2012 as iaudata
+        from .astropyconst13 import codata2010 as codata_context
```

Comment:
In principle, one could try importing from the given module name. In that case, inside those modules `codata` and `iaudata` have to be defined.

But fine to do that later (maybe leave a comment?)

Reply:
I think this is a good idea. Should we only have `codata` and `iaudata` defined inside `astropyconst13` and `astropyconst20`? 

We can also simplify the contents of `config.py` to:
```
import importlib
import astropy

phys_version = astropy.physical_constants.get()
astro_version = astropy.astronomical_constants.get()

codata = importlib.import_module('.constants.' + phys_version, 'astropy')
iaudata = importlib.import_module('.constants.' + astro_version, 'astropy')
```

Reply:
And here I thought it would be just a small item - this now looks like it makes things considerably easier!